helm/cas-bciers/templates/backend/job/reset-database.yaml

```diff
@@ -0,0 +1,88 @@
+{{- if not (hasSuffix "-prod" .Release.Namespace)}}
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+{{- include "cas-bciers.labels" . | nindent 4 }}
+    component: backend
+  name: {{ template "cas-bciers.fullname" . }}-reset-database
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-upgrade
+spec:
+  parallelism: 1
+  completions: 1
+  template:
+    metadata:
+      name: {{ template "cas-bciers.fullname" . }}-reset-database
+      labels:
+{{- include "cas-bciers.labels" . | nindent 8 }}
+        component: backend
+    spec:
+      activeDeadlineSeconds: 180
+      containers:
+        - name:  {{ template "cas-bciers.fullname" . }}-reset-database
+          image: postgres:14.6-alpine
+          env:
+            - name: PGUSER
+              valueFrom:
+                secretKeyRef:
+                  key: user
+                  name: cas-obps-postgres-pguser-postgres
+            - name: PGPASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: password
+                  name: cas-obps-postgres-pguser-postgres
+            - name: PGDATABASE
+              valueFrom:
+                secretKeyRef:
+                  key: dbname
+                  name: cas-obps-postgres-pguser-postgres
+            - name: PGPORT
+              valueFrom:
+                secretKeyRef:
+                  key: port
+                  name: cas-obps-postgres-pguser-postgres
+            - name: PGHOST
+              valueFrom:
+                secretKeyRef:
+                  key: host
+                  name: cas-obps-postgres-pguser-postgres
+            - name: APP_USER
+              valueFrom:
+                secretKeyRef:
+                  key: user
+                  name: cas-obps-postgres-pguser-registration
+{{- if (hasSuffix "-test" .Release.Namespace)}}
+          command:
+            - /usr/bin/env
+            - bash
+            - -c
+            - |
+              psql -c "create schema if not exists preserve;";
+              psql -c "create table preserve.user (first_name text, last_name text, position_title text, email text, phone_number text, user_guid uuid, business_guid uuid, bceid_business_name text, app_role_id text);";
+              psql -c "insert into preserve.user (select * from erc.user);";
```

Comment:
This could be done as
```
create table preserve.user as (select * from erc.user);
```

This way if we alter the table, we're not breaking this script

Reply:
Thanks, I was basing it off the `cas-reg1` chart, happy to hear any refinements since raw SQL isn't my strong suit. Modified in 5bb56e7.